consensus/misc/rewards.go

```diff
@@ -62,3 +72,35 @@ func FindMinDenominations(reward *big.Int) map[uint8]uint8 {
 
 	return denominationCount
 }
+
+// calculateExchangeRate calculates the exchange rate for the block
+func CalculateExchangeRate(parent *types.WorkObject) (*big.Int, *big.Int, *big.Int, error) {
+	qiToQuai := parent.Header().QiToQuai() //Units of Qi
```

Comment:
i thought we are going to use the values from the prime header. aka use the prime terminus for these values, unless we adust these values on every zone block, idk how we can use them